proofs/goal.ml

```diff
@@ -65,6 +65,7 @@ module V82 = struct
     let evi = { Evd.evar_hyps = hyps;
 		Evd.evar_concl = concl;
 		Evd.evar_filter = Evd.Filter.identity;
+                Evd.evar_abstract_arguments = Evd.Abstraction.identity;
```

Comment:
there is something wrong with the indentation

Reply:
The surrounding lines used tabs, I changed them to spaces.